dangerzone/util.py

```diff
@@ -2,6 +2,18 @@
 import platform
 import subprocess
 import sys
+from typing import Optional
+
+import appdirs
+
+
+def get_config_dir() -> str:
+    return appdirs.user_config_dir("dangerzone")
+
+
+# By default we create temporary dirs under /tmp, which is why we return None.
+def get_tmp_dir() -> Optional[str]:
+    return None
```

Comment:
I'd only add here s note that this function exists so it can be mocked. Otherwise, having this seems to make little sense.

Reply:
Sure, that's sensible. I'll add this note and merge the PR.